docs/user-manual/en/address-model.md

```diff
@@ -996,3 +997,14 @@ queues created on the matching address. Defaults to 0. Read more about
 `enable-metrics` determines whether or not metrics will be published to any
 configured metrics plugin for the matching address. Default is `true`. Read more
 about [metrics](metrics.md).
+
+`enable-ingress-timestamp` determines whether or not the broker will add its
+timestamp to messages sent to the matching address. When `true` the broker will
+treat AMQP and non-AMQP messages differently. For AMQP messages the broker will
+add a *message annotation* named `x-opt-ingress-timestamp`. For core messages
+(used by the core and OpenWire protocols) the broker will add a property named
+`_AMQ_INGRESS_TIMESTAMP`. For STOMP messages the broker will add a frame header
+named `ingress-timestamp`. The value will be a 64-bit
+[epoch](https://en.wikipedia.org/wiki/Unix_time) timestamp with
+[*nano*second](https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#nanoTime--)
```

Comment:
Per earlier comment, this doesnt match what the value sent does, as System.nanoTime() isnt epoch based.